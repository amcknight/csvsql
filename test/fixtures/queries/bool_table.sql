SELECT name, NOT isCool AS awesomeButNotCool
FROM test/fixtures/tables/cool
WHERE isAwesome
