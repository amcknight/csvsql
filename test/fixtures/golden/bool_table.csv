name, awesomeButNotCool
Tom, True
July, False
Andrew, True
